src/metabase/lib/join.cljc

```diff
@@ -480,7 +496,7 @@
   (m/find-first lib.types.isa/primary-key?
                 (lib.metadata.calculation/visible-columns query stage-number x)))
 
-(mu/defn ^:private fk-column :- [:maybe lib.metadata/ColumnMetadata]
+(mu/defn ^:private fk-column-for :- [:maybe lib.metadata/ColumnMetadata]
```

Comment:
@bshepherdson suggested this name change in my previous PR, but I missed it so I did it here instead.